hspec-core/src/Test/Hspec/Core/Extension/Spec.hs

```diff
@@ -0,0 +1,10 @@
+module Test.Hspec.Core.Extension.Spec (
+  mapItems
+) where
+
+import           Prelude ()
+
+import           Test.Hspec.Core.Spec
+
+mapItems :: (Item a -> Item b) -> SpecWith a -> SpecWith b
```

Comment:
Seems like a better home for this is in hspec-core/src/Test/Hspec/Core/Extension/Item.hs

Reply:
We have `Spec.mapItems` and `Tree.mapItems`, that's why they are in their respective modules.  I added some rudimentary documentation to hopefully outline this: https://github.com/hspec/hspec/pull/868/files

Reply:
good point